python/geo2png.py

```diff
@@ -0,0 +1,121 @@
+# trafo -> png
+# Original source https://github.com/vrandezo/wikidata-analytics/blob/6896d639bd124eec7571f1179f27ae2b6bfaa435/geo2png.py
```

Comment:
Well, the original is at http://tools.wmflabs.org/wikidata-analysis/pre2015/map/ (not added by me)
I am fine with waiting for @vrandezo though...